app/brave_generated_resources.grd

```diff
@@ -364,6 +364,11 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         Block elements
       </message>
 
+      <!-- Email aliases context menu -->
+      <message name="IDS_NEW_EMAIL_ALIAS" desc="Message for context menu that enables the Email Alias generator UI for the page">
```

Comment:
is this used someplace? Also shouldn't this be in strings for the component?

Reply:
This was accidentally left in from the context menu which will be added in a follow-up. Removed.